### PR TITLE
update for discover-scu17 mkl and docs

### PR DESCRIPTION
**Pull request by ashley314**
### Summary

Updates needed to discover-scu17 site configuration for release/1.8.0

### Testing

Describe the testing done for this PR.

### Systems affected

discover-scu17 intel

### Issue(s) addressed

Addresses https://github.com/JCSDA/spack-stack/issues/1278

### Checklist
- [x] This PR addresses one issue/problem/enhancement, or has a very good reason for not doing so.
- [x] These changes have been tested on the affected systems and applications.
- [x] All dependency PRs/issues have been resolved and this PR can be merged.

### CODE REVIEW

File: doc/source/PreConfiguredSites.rst [hunk 1]
```diff
@@ -118,16 +118,30 @@ NASA Discover SCU16
...
-
 .. code-block:: console
 
    module purge
    module use /discover/swdev/gmao_SIteam/modulefiles-SLES12
    module use /discover/swdev/jcsda/spack-stack/scu16/modulefiles
    module load miniconda/3.9.7
-   module load ecflow/5.8.4
 
+For ``spack-stack-1.8.0`` with Intel, load the following modules after loading miniconda:
```
Comment (climbfuji) [inline doc/source/PreConfiguredSites.rst]: This doesn't go into readthedocs anymore, see nautilus/narwhal - instead, please update https://github.com/JCSDA/spack-stack/wiki/spack%E2%80%90stack%E2%80%901.8.0-release-documentation

File: doc/source/PreConfiguredSites.rst [hunk 2]
```diff
@@ -137,15 +151,29 @@ NASA Discover SCU17
...
 The following is required for building new spack environments with any supported compiler on this platform.
 
-**NEEDS UPDATING**
-
 .. code-block:: console
 
    module purge
    module use /discover/swdev/gmao_SIteam/modulefiles-SLES15
    module use /discover/swdev/jcsda/spack-stack/scu17/modulefiles
-   module load ecflow/5.11.4
```
Comment (climbfuji) [inline doc/source/PreConfiguredSites.rst:141]: same as above - only the parts that are needed to build new spack environments go here, whatever is needed to load existing spack environments (including the module purge /module use statements that maybe the same as the ones that remain in readthedocs) go into the wiki

**Review by climbfuji — CHANGES_REQUESTED** [review]

File: doc/source/PreConfiguredSites.rst (hunk 2 above)
Comment (ashley314) [reply doc/source/PreConfiguredSites.rst:141]: Ah will do!

**Review by climbfuji — APPROVED** [review]
Looks good. Thanks for updating https://github.com/JCSDA/spack-stack/wiki/spack%E2%80%90stack%E2%80%901.8.0-release-documentation too

**Review by srherbener — APPROVED** [review]
Thanks for finding and fixing the SCU17 intel typo! And thanks for building the Discover environments.

**Pull request merged**